bin/scripts/capture-screenshots

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+const puppeteer = require('puppeteer');
+const del = require('del');
+
+const DEST_PATH = path.join(__dirname, '../../screenshots');
+
+const FILE_FORMAT = 'png';
+
+const SIZES_TO_CAPTURE = {
+    small: {
+        width: 414,
+        height: 736
+    },
+    medium: {
+        width: 768,
+        height: 1024
+    },
+    large: {
+        width: 1024,
+        height: 768
+    }
+};
+
+function urlPathToSlug(urlPath, size) {
+    const urlSlug = urlPath.substring(1).replace('/', '-');
+    const dateSlug = new Date().toISOString();
+    const sizeSlug = size ? `${size.width}x${size.height}` : '';
+    const parts = [urlSlug, sizeSlug, dateSlug].filter(_ => _).join('-');
+    return `${parts}.${FILE_FORMAT}`;
+}
+
+del.sync([`${DEST_PATH}/*.${FILE_FORMAT}`]);
+puppeteer.launch({ dumpio: true, args: ['--no-sandbox', '--disable-setuid-sandbox'] }).then(browser => {
+    function capturePageSizes(urlPath, size) {
+        return browser.newPage().then(page => {
+            return page.goto(`https://test.blf.digital${urlPath}`, { waitUntil: 'networkidle' }).then(() => {
+                const screenshotFilename = urlPathToSlug(urlPath, size);
+                const screenshotFullPath = `${DEST_PATH}/${screenshotFilename}`;
+                return page.setViewport(size).then(() => {
+                    return page.screenshot({ path: screenshotFullPath, fullPage: true }).then(() => {
+                        assert(fs.existsSync(screenshotFullPath));
+                        console.log(`${screenshotFullPath} saved 🎉 `);
+                        return page.close();
+                    });
+                });
+            });
+        });
+    }
+
+    const promises = [].concat(
```

Comment:
This may be me not understanding this syntax but why do this over just listing a regular array?